errors/src/lib.rs

```diff
@@ -14,6 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
+#![recursion_limit = "256"]
```

Comment:
Curious what triggered the compiler limit? Do we need to worry about hitting this in the future?